tests2/omit.test.ts

```diff
@@ -73,13 +73,13 @@ const fieldsMatrix = [
 
 function makeList({
   fields,
-  isFilterable,
-  isOrderable,
```

Comment:
I edited this file as I was reviewing tests to understand how this was missed.

We aren't testing this functionality well.
For now I will move forward with the fixes as is, but I do have some new tests in the pipeline to add coverage for this regression and others.